Makefile

```diff
@@ -503,14 +503,14 @@ gen-doc =	\
 		else \
 			cd tools/doc && node ../../$(NPM) install; \
 		fi;\
-	[ -x $(NODE) ] && $(NODE) $(gen-json) || node
+	[ -x $(NODE) ] && $(NODE) $(1) || node $(1)
 
 out/doc/api/%.json: doc/api/%.md
-	$(gen-doc) $(gen-json)
+	$(call gen-doc, $(gen-json))
 
 # check if ./node is actually set, else use user pre-installed binary
```

Comment:
I guess this comment is misused here. We have copy of this comment above which is probably should be placed right above `gen-doc` definition.